drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -264,6 +264,11 @@ enum wifi_nrf_status wifi_nrf_fmac_dev_add_zep(struct wifi_nrf_drv_priv_zep *drv
 	struct wifi_nrf_ctx_zep *rpu_ctx_zep = NULL;
 	struct wifi_nrf_fmac_fw_info fw_info;
 	void *rpu_ctx = NULL;
+#if defined(CONFIG_BOARD_NRF7000) || defined(CONFIG_BOARD_NRF7001)
+	enum op_band op_band = BAND_24G;
```

Comment:
How will this work with OTP programming? We need to document the requirements somewhere?